src/aap_eda/api/serializers/auth.py

```diff
@@ -14,9 +14,30 @@
 
 from rest_framework import serializers
 
+from aap_eda.services.auth import jwt_access_token, validate_jwt_token
+from aap_eda.services.exceptions import InvalidTokenError
+
 
 class LoginSerializer(serializers.Serializer):
     username = serializers.CharField(required=True)
     password = serializers.CharField(
         required=True, style={"input_type": "password"}
     )
+
+
+class RefreshTokenSerializer(serializers.Serializer):
```

Comment:
@bzwei This needs 2 methods if its a Serializer

https://github.com/encode/django-rest-framework/blob/8e304e1adbb0f99f91a15ed3abd379104bba3b89/rest_framework/serializers.py#L170
https://github.com/encode/django-rest-framework/blob/8e304e1adbb0f99f91a15ed3abd379104bba3b89/rest_framework/serializers.py#L173

Reply:
Those methods are not used. `LoginSerializer` in the same py file does not implement them either.